1/styles/scss/_accordion.scss

```diff
@@ -0,0 +1,68 @@
+.accordion{
+  padding-top: 100px;
+
+  .accordion-item{
+
+    &:last-child > .accordion-item-header{
+      border-bottom: 1px solid $accent;
+    }
+
+    &.show:last-child > .accordion-item-header{
+      border-bottom: 1px solid transparent;
+    }
+    &-header {
+
```

Comment:
Same thing, clean up your SCSS, you have to decide if you have a line separator after the selector or not, and store it consistently through the code